build

```diff
@@ -180,7 +180,7 @@ just_compile_elixir() {
     mix local.rebar --if-missing --force
     mix local.rebar rebar3 "${PWD}/rebar3" --if-missing --force
     # mix local.hex --if-missing --force
-    mix local.hex 2.0.6 --if-missing --force
+    mix local.hex 2.2.1 --if-missing --force
```

Comment:
align with Makefile

Reply:
in theory, the commented line above this one should work for both this script and Makefile

the reason we have pinned a specific version is that once in the past it happened that a hex release was buggy, so we rolled back to a previous release